clients/web/test/spec/dataSpec.js

```diff
@@ -893,7 +893,7 @@ describe('Test FileModel static upload functions', function () {
                 method: 'GET',
                 data: {
                     folderId: folder.get('_id'),
-                    text: filename
```

Comment:
Does this break backward compatibility ?

Reply:
I don't think so, but I will need to look at it again